packages/modules/src/modules/epics/ContributionsEpicChoiceCards.tsx

```diff
@@ -19,6 +19,19 @@ const frequencyChoiceCardGroupOverrides = css`
     }
 `;
 
+const hideChoiceCardGroupLegend = css`
+    legend {
+        border: 0;
+        clip: rect(0 0 0 0);
+        height: 1px;
+        margin: -1px;
+        overflow: hidden;
+        padding: 0;
+        position: absolute;
+        width: 1px;
+    }
```

Comment:
There is a `visuallyHidden` const in source we could potentially use? https://github.com/guardian/source/blob/867913e361c3eca58b20cf313d1a02d03d4c911a/packages/@guardian/source-foundations/src/accessibility.ts#L12

imported like:

```ts
import { visuallyHidden } from '@guardian/src-foundations/accessibility
```

and then

```
legend {
  ${visuallyHidden}
}
```

Reply:
Also, I wonder more generally if it might be useful for source to have a `hideLabel` option here for the `ChoiceCardGroup`, just like it does for buttons

Reply:
Sounds like source is adding that! 🎉

Reply:
Made that update so we use the const from source and once the `hideLabel` prop is available will update this code in another small PR